de/translation.json

```diff
@@ -317,7 +315,7 @@
   "Common.LevelNth": "Level {{number}}",
   "Common.Location": "Standort",
   "Common.Lore": "Lore",
-  "Common.LoreBook": "🦘 Lore Book",
+  "Common.LoreBook": "Lore Buch",
```

Comment:
Not sure if translating Lore would be desirable here, but it wasn't translated for `Common.Lore` so we're keeping consistency

Reply:
Went for "Lorebuch"